src/Evolution/DgSubcell/SliceVariable.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+#include <cstddef>
+
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+#include "Evolution/DgSubcell/SliceData.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace evolution::dg::subcell {
+/// @{
+/*!
+ * \brief Slice a Variables object on subcell mesh for a given direction and
+ * slicing depth (number of ghost points).
+ *
+ * This is essentially a wrapper of
+ * `evolution::dg::subcell::detail::slice_data_impl` function, but note that the
```

Comment:
I don't know how great it is to refer to a `detail` function in the documentation. It's a detail for a reason, not a public interface. Isn't the contrast to the other `slice_data`?